roles/java/vars/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # vars file for roles/java
 jdk_tar_file: "OpenJDK11U-jdk_x64_linux_{{ dependencies_version.jdk }}_{{ dependencies_version.jdk_arch }}.tar.gz"
-jdk_folder: "openjdk-{{ dependencies_version.jdk }}+{{ dependencies_version.jdk_arch }}"
+jdk_folder: "openjdk-{{ dependencies_version.jdk }}_{{ dependencies_version.jdk_arch }}"
```

Comment:
this is worrying me as it may means that playbook fails if user points to a version where the content of the archive has + or any unexpected character...
Do we have any idea how much this change from openjdk is stable?